data/dev.geopjr.tooth.gschema.xml

```diff
@@ -1,14 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schemalist>
 
+  <enum id="dev.geopjr.tooth.ColorScheme">
+    <value value="0" nick="system"/>
+    <value value="1" nick="light"/>
+    <value value="2" nick="dark"/>
+  </enum>
+
```

Comment:
Here I add an enumeration to the gschema, which takes certain values and nick to set it accordingly